groups.json

```
@@ -7,6 +7,10 @@
     "display": "Bloom filter",
     "description": "Operations on the Bloom filter data type"
   },
+  "search": {
+    "display": "Search",
+    "description": "Vector Similarity Search engine optimized for AI-driven workloads"
```

```suggestion
    "description": "Vector Similarity Search engine optimized for AI-driven workloads"
```
The intention is to make this a generic search module? We should probably not highlight just vectory search right now.